docs/site/migration/models/mixins.md

```diff
@@ -116,13 +121,30 @@ the mixin.
 
 In LoopBack 4, a developer is able to create a model property mixin by:
 
+- creating a base model class which extends `Entity`
 - placing the mixin class factory function in a separate file
 - generating a model using the CLI as usual
 - adjusting the model file to make use of the mixin class factory function
 
+### Defining A BaseModel Class Which Extends Entity
+
+Let's define a base model class `BaseModel` in **src/models/baseModel.ts**. It
+will be used an input to the mixin later.
+
+{% include code-caption.html content="src/models/baseModel.ts" %}
+
+```ts
+import {Entity} from '@loopback/repository';
+export class BaseModel extends Entity {}
```

Comment:
I find it confusing that the class is called `BaseModel` when it's extending from `Entity`. What if the user wants to implement a mixin that can be applied to `Model` subclasses too, not only to `Entity` descendants? 

I am proposing to use the name `BaseEntity` instead.